governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+- The WBTC-C [Stability Fee (SF)](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%, from 5.61% to *5.54%*.
+
+
+### Smart Burn Engine Parameter Optimization
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-3/22876/2)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-3/22876) "Smart Burn Engine - Transaction Analysis & Parameter Reconfiguration Update #3"
+
+If this executive proposal passes, following the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SBE parameters will be changed:
+
+- The [bump (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) parameter will be increased by 10,000, from 20,000 to *30,000*.
+- The [hop (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) parameter will be increased by 9,460, from 6,308 to *15,768*.
+
+
+### Approve RWA Foundation Dao Resolution
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/dao-resolution-rwa-foundation-service-provider-changes/22866/2)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/dao-resolution-rwa-foundation-service-provider-changes/22866) "DAO Resolution - RWA Foundation Service Provider Changes"
+
+If this executive proposal passes, the following RWA Foundation Dao Resolution to change the service provider will be approved:
+
+- The [DAO Resolution](https://gateway.pinata.cloud/ipfs/QmPiEHtt8rkVtSibBXMrhEzHUmSriXWz4AL2bjscq8dUvU) with IPFS hash `QmPiEHtt8rkVtSibBXMrhEzHUmSriXWz4AL2bjscq8dUvU` will be approved.
+
+
+### DAI transfer to cover Andromeda Legal expenses
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577/4)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577) "Project Andromeda - Legal Expenses II"
+
+If this executive proposal passes, the following transfer will be performed to cover Andromeda (RWA015-A) Legal expenses:
+
+- *201,738 DAI* will be transfered to [0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767](https://etherscan.io/address/0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767).
+
+
+### Cease MKR and DAI Vest Streams for Steakhouse and TechOps
+
+*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\
+
+If this executive proposal passes, in line with the recent amendments in the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a), the following DSS Vest Streams will be halted by calling the Yank function:
+
+- Steakhouse *DAI* stream *ID 19*
+- TechOps *DAI* stream *ID 18*
+- Steakhouse *MKR* stream *ID 32*
+- TechOps *MKR* stream *ID 33*
+
+
+### Set up MKR and DAI Vest Streams for Ecosystem Facilitator
+
+*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\
```

Comment:
```suggestion
*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)
```

The backslash is being rendered in the rich diff, this is presumably not desirable